cocotb/result.py

```diff
@@ -73,14 +73,14 @@ def create_error(obj, msg):
     simply to avoid too many levels of nested `try/except` blocks.
 
     .. deprecated:: 1.3
-        Use ``TestError(msg)`` directly instead of this function.
+        Use ``raise TestError(msg)`` directly instead of this function.
 
     Args:
         obj: Object with a log method.
         msg (str): The log message.
     """
     warnings.warn(
-        "``create_error`` is deprecated - use ``TestError(msg)`` (or any other "
+        "``create_error`` is deprecated - use ``raise TestError(msg)`` (or any other "
```

Comment:
This change isn't correct

Reply:
Oh okay, it looked like an inconsistency.

Reply:
See above- the other change is correct